zio-cache/shared/src/main/scala/zio/cache/ManagedCache.scala

```diff
@@ -69,43 +69,43 @@ object ManagedCache {
    * Constructs a new cache with the specified capacity, time to live, and
    * lookup function.
    */
-  def make[Key, Environment, Error, Value](
+  def make[Key, Env, Error, Value](
     capacity: Int,
     timeToLive: Duration,
-    lookup: ManagedLookup[Key, Environment, Error, Value]
-  ): URManaged[Environment, ManagedCache[Key, Error, Value]] =
+    lookup: ManagedLookup[Key, Env, Error, Value]
+  ): UManaged[ManagedCache[Key, Env, Error, Value]] =
     makeWith(capacity, lookup)(_ => timeToLive)
 
   /**
    * Constructs a new cache with the specified capacity, time to live, and
    * lookup function, where the time to live can depend on the `Exit` value
    * returned by the lookup function.
    */
-  def makeWith[Key, Environment, Error, Value](
+  def makeWith[Key, Env, Error, Value](
     capacity: Int,
-    managedLookup: ManagedLookup[Key, Environment, Error, Value]
-  )(timeToLive: Exit[Error, Value] => Duration): URManaged[Environment, ManagedCache[Key, Error, Value]] =
+    managedLookup: ManagedLookup[Key, Env, Error, Value]
+  )(timeToLive: Exit[Error, Value] => Duration): UManaged[ManagedCache[Key, Env, Error, Value]] =
     makeWith(capacity, managedLookup, Clock.systemUTC())(timeToLive)
 
   //util for test because it allow to inject a mocked Clock
-  private[cache] def makeWith[Key, Environment, Error, Value](
+  private[cache] def makeWith[Key, Env, Error, Value](
     capacity: Int,
-    managedLookup: ManagedLookup[Key, Environment, Error, Value],
+    managedLookup: ManagedLookup[Key, Env, Error, Value],
     clock: Clock
-  )(timeToLive: Exit[Error, Value] => Duration): URManaged[Environment, ManagedCache[Key, Error, Value]] =
+  )(timeToLive: Exit[Error, Value] => Duration): UManaged[ManagedCache[Key, Env, Error, Value]] =
     ZManaged.make(buildWith(capacity, managedLookup, clock)(timeToLive))(_.invalidateAll)
 
-  private def buildWith[Key, Environment, Error, Value](
+  private def buildWith[Key, Env, Error, Value](
     capacity: Int,
-    managedLookup: ManagedLookup[Key, Environment, Error, Value],
+    managedLookup: ManagedLookup[Key, Env, Error, Value],
     clock: Clock
-  )(timeToLive: Exit[Error, Value] => Duration): URIO[Environment, ManagedCache[Key, Error, Value]] =
```

Comment:
I'm not sure why we were returning a `URIO[Environment, ...]` 🤔

Do we want to keep this `Environment`? If yes, we'll need two envs for this function.